.github/workflows/ci.yml

```diff
@@ -25,7 +25,18 @@ jobs:
       - uses: actions/checkout@v2
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@1.1.0
-
+  
+  shfmt:
+    name: shfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: reviewdog/action-shfmt@v1.0.2
+        with:
+          filter_mode: "nofilter"
+          fail_on_error: "true"
+          shfmt_flags: "-i 2 -ci -bn"
```

Comment:
Flags from https://google.github.io/styleguide/shellguide.html#s5-formatting

Reply:
Couldn't find the flags in the doc you reference. I found [this](https://github.com/mvdan/sh/blob/3c0f553b3180d01710580241fcbf5f89a87f48d3/cmd/shfmt/shfmt.1.scd#examples) which seems to match the flags you chose.

I guess you are planning to submit some scripts :-)

Reply:
yeah a script so we can locally run shfmt with the same setup would be great :) Is there maybe a config file so we can just run shfmt?